elections/2023-TOC/election_desc.md

```diff
@@ -0,0 +1,27 @@
+# 2023 Voters Guide - Knative TOC Election
+
+The role of this election is to fill the one (1) seat due for election this year on the [Knative Technical Oversight Committee](https://github.com/knative/community/blob/master/TECH-OVERSIGHT-COMMITTEE.md). The elected member will serve a two (2) year term.
+
+This election will shape the future of Knative as a community and project.
+While WGs help shape the individual direction of the project, the
```

Comment:
Couple of small nits... "WG" is not spelled out anywhere in this document, so I suggest writing it out fully. Also, "individual direction of the project" is a little unclear to me. Here is a suggestion that may be better.
```suggestion
While working groups help shape the direction of individual subprojects, the
```